pipeline/splicing/calcVarPriors.py

```diff
@@ -909,22 +993,57 @@ def getClosestSpliceSiteScores(variant, deNovoOffset, donor=True, deNovo=False,
         closestSpliceBounds = getVarSpliceRegionBounds(variant, donor=donor, deNovo=deNovo)
         exonName = closestSpliceBounds["exonName"]
     if donor == True:
-        refSeq = getFastaSeq(varChrom, closestSpliceBounds["donorStart"], closestSpliceBounds["donorEnd"])
+        if getVarStrand(variant) == "+":
+            refSeq = getFastaSeq(varChrom, closestSpliceBounds["donorStart"], closestSpliceBounds["donorEnd"], plusSeq=True)
+            # splice site is 3 bp to the right of donor Start (+3 because plus strand numbering increases from left to right)
+            genomicSplicePos = closestSpliceBounds["donorStart"] + 3
```

Comment:
It is 3 bp to the right because the donor start position is 3 bp to the left of the exon end, so it's just getting the exon end position. I added a comment, but if you think it's not detailed enough I can expand it